sspl_test/sspl_start_checker

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3.6
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+
+import sys
+import os
+import json
+import socket
+
+# Adding sspl and sspl_test path
+test_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+os.sys.path.append(os.path.join(test_path))
+from cortx.utils.message_bus import MessageBus
+from cortx.utils.message_bus import MessageConsumer
+
+
+def process_msg(body):
```

Comment:
what this file is mainly for ? usecase

Reply:
This file is for waiting till SSPL adds `SSPL-LL service has started successfully` message into message bus, This message indicated that SSPL is initialized successfully and ready to accept requests. After that tests cases can be executed